docs/src/main/paradox/producer.md

```diff
@@ -18,49 +18,76 @@ Scala
+In addition to programmatic construction of the `ProducerSettings` (@scaladoc[API](akka.kafka.ProducerSettings)) it can also be created from configuration (`application.conf`). 
+
+When creating `ProducerSettings` with the `ActorSystem` (@scaladoc[API](akka.actor.ActorSystem)) settings it uses the config section `akka.kafka.producer`. The format of these settings files are described in the [Typesafe Config Documentation](https://github.com/lightbend/config#using-hocon-the-json-superset).
 
 @@ snip [flow](../../../../core/src/main/resources/reference.conf) { #producer-settings }
 
-`ProducerSettings` (@scaladoc[API](akka.kafka.ProducerSettings)) can also be created from any other `Config` (@scaladoc[API](com.typesafe.config.Config)) section with the same layout as above.
+`ProducerSettings` (@scaladoc[API](akka.kafka.ProducerSettings)) can also be created from any other `Config` section with the same layout as above.
 
 See @javadoc[KafkaProducer API](org.apache.kafka.clients.producer.KafkaProducer) and @javadoc[ProducerConfig API](org.apache.kafka.clients.producer.ProducerConfig) for more details regarding settings.
 
 ## Producer as a Sink
 
 `Producer.plainSink` 
 (@scala[@scaladoc[Producer API](akka.kafka.scaladsl.Producer)]@java[@scaladoc[Producer API](akka.kafka.javadsl.Producer)]) 
-is the easiest way to publish messages. The sink consumes `ProducerRecord` (@javadoc[API](org.apache.kafka.clients.producer.ProducerRecord)) elements which contains a topic name to which the record is being sent, an optional partition number, and an optional key and value.
+is the easiest way to publish messages. The sink consumes the Kafka type `ProducerRecord` (@javadoc[API](org.apache.kafka.clients.producer.ProducerRecord)) which contains 
+
+1. a topic name to which the record is being sent, 
+1. an optional partition number, 
+1. an optional key, 
+1. and a value.
 
 Scala
 : @@ snip [plainSink](../../test/scala/sample/scaladsl/ProducerExample.scala) { #plainSink }
-  The materialized value of the sink is a `Future[Done]` which is completed with `Done` when the stream completes or with exception if an error occurs.
+  The materialized value of the sink is a `Future[Done]` which is completed with `Done` when the stream completes, or with with an exception in case an error occurs.
 
 Java
 : @@ snip [plainSink](../../test/java/sample/javadsl/ProducerExample.java) { #plainSink }
-  The materialized value of the sink is a `CompletionStage<Done>` which is completed with `Done` when the stream completes or with exception if an error occurs.
+  The materialized value of the sink is a `CompletionStage<Done>` which is completed with `Done` when the stream completes, or with an exception in case an error occurs.
+
+### Connecting a Producer to a Consumer
```

Comment:
This section is already in the Consumer page. We should link to there, rather than duplicating the contents. Also move a `@@@note` from below about the failure after publishing and before commiting to the "Connecting Producer and Consumer" section on the Consumer page.